frontend/src/scenes/insights/insightLogic.test.ts

```diff
@@ -304,7 +302,7 @@ describe('insightLogic', () => {
         })
     })
 
-    describe.skip('as dashboard item', () => {
```

Comment:
it's been skipped for a year, only keep the tests that currently pass when unskipped